cypress/integration/components/generator.spec.js

```diff
@@ -0,0 +1,39 @@
+describe('Generator component', () => {
+    beforeEach(() => {
+        cy.visit('/generator');
+    });
+
+    it('has generated a blob URL', () => {
+        cy.get('.request-transport-medium-chooser')
+            .contains('Fax')
+            .click();
+
+        cy.contains('Download PDF')
+            .should('not.have.class', 'disabled')
+            .should('have.attr', 'href')
+            .and('match', /^blob:https?:\/\/[\S]+?\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/);
+    });
+
+    it('did not load pdfworker for email', () => {
+        cy.window().then(win => {
+            expect(win.pdfWorker).to.be.undefined;
+        });
+    });
+
+    it('did load pdfworker for fax', () => {
+        cy.get('.request-transport-medium-chooser')
+            .contains('Fax')
+            .click();
+
+        cy.window().then(win => {
+            expect(win.pdfWorker).not.to.be.undefined;
+            // somehow it does not work with this: expect(win.pdfWorker).to.satisfy((o) => o instanceof Worker);
+            // "console.log(win.pdfWorker instanceof Worker)" logs false...
+            // time to 🦆 type...
```

Comment:
How do you like emojis in our comments, @baltpeter? 😜

Reply:
Big fan!